src/server/scripts/Kalimdor/TempleOfAhnQiraj/temple_of_ahnqiraj.h

```diff
@@ -39,6 +39,7 @@ enum DataTypes
     DATA_OURO               = 13,
     DATA_OURO_SPAWNER       = 14,
     DATA_BUG_TRIO_DEATH     = 15,
+    DATA_HUHURAN            = 16,
```

Comment:
Define this one below all of them, it will be changed by my pr after all (to minimize merge conflicts)

Reply:
sure